lib/chef-dk/command/generator_commands/base.rb

```diff
@@ -85,7 +85,13 @@ def setup_context
         def have_git?
           path = ENV["PATH"] || ""
           paths = path.split(File::PATH_SEPARATOR)
-          paths.any? {|bin_path| File.exist?(File.join(bin_path, "git#{RbConfig::CONFIG['EXEEXT']}"))}
+          exts = [RbConfig::CONFIG['EXEEXT']]
+          exts.concat(ENV["PATHEXT"].split(";")) unless ENV["PATHEXT"].nil?
+          paths.any? do |bin_path|
+            exts.any? do |ext|
+              File.exist?(File.join(bin_path, "git#{ext}"))
```

Comment:
Shouldn't need to. `any?` breaks at the first iteration that returns `true` and returns `false` otherwise: http://stackoverflow.com/a/20688735/319074